### DISCUSSION TITLE
prismaClientInitializationError when deployed to Netlify

### DISCUSSION THREAD

**Discussion by rajeshwarmanthena** [prisma/prisma#23162] [Q&A]
### Question

{"message":"GET Error","err":{"name":"PrismaClientInitializationError","clientVersion":"5.8.1"}}

I deployed my Next.js website to netlify. I connected my website to mongoDB with prisma ORM , but my deployed site is unable to connect to mongodb. 
I checked my environmental variable it's okay, when I run my source code in my IDE it connects well with Mongodb.
My network access of data base and data access are also fine.
I TRIED EVERYTHING BUT couldn’t solve it.
https://connect-donor.netlify.app/ (THIS IS WHERE I AM GETTING ERROR)

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
generator client {
  provider = "prisma-client-js"
  binaryTargets = ["native", "rhel-openssl-1.0.x"]
}

datasource db {
  provider = "mongodb"
  url      = env("DATABASE_URL")
}

model Donor {
  id         String @id @default(auto()) @map("_id") @db.ObjectId
  firstName  String
  lastName   String
  phone      String
  email      String @unique
  bloodGroup String
  age        Int
  address    String
  state      String
  city       String
  gender     String
}

```

```ts
// Add any relevant Prisma Client queries here
```

- OS: macOS
- Database: mongoDB
- Node.js version: v21.6.1

- Run `prisma -v` to see your Prisma version and paste it

**Comment by janpio** [comment]
The link you shared loads fine for me.

What is the actual error getting logged in the function logs? What you shared is just the "title" of the error, not the actual error message itself.

**Reply by rajeshwarmanthena** [reply]
it loads fine!!! 
But it is not connecting to MongoDB

**Comment by iLoveHanekawa** [comment]
Please ensure its not caused by any of the following reasons:

- The provided credentials for the database are invalid
- There is no database server running under the provided hostname and port
- The port that the query engine HTTP server wants to bind to is already taken
- A missing or inaccessible environment variable
- The query engine binary for the current platform could not be found (generator block)

[Error docs](https://www.prisma.io/docs/orm/reference/error-reference#prismaclientinitializationerror)

im sorry if this is an impolite suggestion but ensure that your db connection string is not referencing a local db and that its actually hosted somewhere (mongodb atlas for instance)

**Comment by ludralph** [comment]
Hi @rajeshwarmanthena 

Did the suggestion above help resolve your issue? If this is still an issue, please do let us know and we will do our best possible to help resolve it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
